targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT/device/system_psoc6.h

```diff
@@ -539,7 +544,11 @@ extern "C" {
 * \addtogroup group_system_config_system_functions
 * \{
 */
-extern void SystemInit(void);
+#if defined(__ARMCC_VERSION)
+    extern void SystemInit(void) __attribute__((constructor));
```

Comment:
remove attribute

Reply:
Fixed in the https://github.com/ARMmbed/mbed-os/pull/10308/commits/d4f433a8cdab56b5ef8994e0c943de0032ae85f8